topics/introduction/tutorials/galaxy-intro-short/tutorial.md

```diff
@@ -0,0 +1,234 @@
+* The actual numbers in front of the files in the history are not important.
+
+What are the results from this filtering tool?
+
+* We could click on the eye icon to view the contents of this output file, but it will not be very informative - we will just see a list of reads.
+* Instead, let's click on the output file name in the History panel. This expands the information about the file.
+* We can see that 1786 low-quality reads were discarded.
+
+![filter1](../../images/filter-fastq1.png)
+
+## Re-run that tool with changed settings
+
+We have now decided that our input reads have to be filtered to an even higher standard.
+* We will change the filter settings and re-run the tool.
+
+> ### {% icon hands_on %} Hands-on: Re-run the tool
+> * In the History panel, find the output file from the first time we ran the filter tool. This file is called "Filter by quality on data 1".
+> * Click on the icon with two arrows - this means "run this tool again".
+> <br>
+> ![rerun](../../images/rerun.png)
+> * This brings up the tool interface in the centre panel.
+> * Change the settings to something even stricter. For example, you might decide you want 80 percent of bases to have a quality of 36 or higher, instead of 35.
+> * Click "Execute".
+{: .hands_on}
+
+View the results:
+*  Click on the output file name to expand the information. (*Note*: not the eye icon.)
+* How many reads were discarded under these new filtering conditions?
+
+You can re-run a tool many times with different settings.
+* Each time you re-run the tool, the new output file will appear at the top of your current history.
+
+## Create a new history
+
+Let's create a new history.
+
+> ### {% icon hands_on %} Hands-on: New history
+> * In the History panel, click on the cog icon.
+> <br>
+> ![cog](../../images/cog.png)
+> <br>
+> * Select "Create New".
+> * Name your history, *e.g.* "Next-analysis"
+> * Press Enter
+{: .hands_on}
+
+This new history does not have any files in it yet.
+
+## Look at all your histories
+
+Where is your first history, called "my-analysis"?
+
+> ### {% icon hands_on %} Hands-on: View histories
+> * In the History panel, click on the "View all histories" icon.
+> <br>
+> ![view-hist](../../images/view-hist.png)
+> <br>
+> * All your histories are displayed here.
+> <br>
+> * Drag a file into your new history:
+> * Click on the FASTQ file in "my-analysis" history
+> * Drag it into the "Next-analysis" history
+> * This makes a copy of the file in the new history
+> * Click "Done"
+> <br>
+> ![view-all-hist](../../images/view-all-hist.png)
+{: .hands_on}
+
+Your main Galaxy window will now show the current history as "Next-analysis", and it will have one file in it.
+
+You can go back into the "View all histories" page and "Switch to" a different history.
+
+
+# Summary
+
+What have we learned?
+
+* The Galaxy interface has tools on the left, viewing pane in the middle, and a history of your data analysis on the right.
```

Comment:
Could you move this list to the key points in the metadata and then merge this section with the conclusion? Then it will be displayed like this:

![screen shot 2018-06-04 at 09 26 49](https://user-images.githubusercontent.com/1842467/40904209-448d0192-67da-11e8-8004-a5dc8bb45cfb.png)

Thanks a lot